tecken/upload/admin.py

```diff
@@ -32,6 +32,9 @@ def download_by_upload(self, obj):
 
 @admin.register(FileUpload)
 class FileUploadAdmin(admin.ModelAdmin):
+    readonly_fields = [
+        "upload",
+    ]
```

Comment:
We never want to change this field, so let's make it readonly. That saves us from Django creating a dropdown with every upload entry ... which is 246k currently and makes loading the page really really rough.